django_project/base/models/location_site.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+"""Site model definition.
+
+"""
+
+from django.contrib.gis.db import models
+from base.models.location_type import LocationType
+
+
+class LocationSite(models.Model):
+    """Location Site model."""
+
+    location_type = models.ForeignKey(
+        LocationType,
+        models.SET_NULL,
+        null=True,
+    )
+    geometry = models.PointField()
```

Comment:
We want to be able to attach data to sites of various types and as such `Location Site` should be able to cater for points, lines, polygons and multipolygons.

Reply:
@cchristelis 
I was following the diagram here : 
![36594631-9a797702-18a7-11e8-9542-88277c3cbdcd](https://user-images.githubusercontent.com/1979569/36765135-5e1241f4-1c62-11e8-884e-9744da5d42b3.png)

So we need to add field for geometry_point, geometry_line, geometry_polygon, and geometry_multipolygon ?